torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -60,11 +65,13 @@
     DistTestCases
 )
 from torch.testing._internal.common_utils import (
+    instantiate_parametrized_tests,
```

Comment:
Typo? `parameterized`? :)

Reply:
It looks like both are used, but parameterized is more common: https://wikidiff.com/parameterize/parametrize

Although, this is already an implemented helper function that I'm just importing and using, so should be okay to leave it as is